docs/runbooks/VirtualMachineCRCErrors.md

```diff
@@ -0,0 +1,49 @@
+# VirtualMachineCRCErrors
+<!-- Edited by machadovilaca, july 2023-->
+
+## Meaning 
+
+Storage Class is incorrectly configured and the usage of a system-wide and
+shared dummy page may cause CRC errors when data is written and read across
+different processes or threads.
+
+## Impact
+
+Cluster may report a huge number of CRC errors and the cluster might experience
+major service outages.
+
+## Diagnosis
+
+1) Get the volume name from the VM:
+
+```bash
+$ kubectl get vm <vm-name> -o jsonpath='{.spec.template.spec.volumes}'
+```
+
+2) Get the storage class name from the volume:
+
+```bash
+$ kubectl get pvc <volume-name> -o jsonpath='{.spec.storageClassName}'
+```
+
+3) Get the storage class configuration:
+
+```bash
+$ kubectl get sc <storage-class-name> -o yaml
+```
+
+4) Check if the storage class has the "krbd:rxbounce" map option:
+
+## Mitigation
+
+Add the "krbd:rxbounce" map option to your storage class configuration.
```

Comment:
Can we add an example?

Reply:
Can you please also share what is the "krbd:rxbounce" map ?

Reply:
added